jobs/SDE/Assignments/SDE-FE-Assignment.md

```diff
@@ -55,17 +52,18 @@ provide option to go back to main page
 
 ### What we care about
 - Architecture:
-    - How clean and reasonable is the separation between modules?
+    - How clean and reasonable is the separation between codes?
+    - How good FE framework specific implementations?
+    - How optimized logic used to solve a problem?
```

Comment:
Can you make this point clearer?

Reply:
could you please explain how? an example would help me to understand what you want...